Podcast/SearchViewController.swift

```diff
@@ -19,6 +19,9 @@ class SearchViewController: ViewController, UISearchControllerDelegate, UITableV
     
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        title = "Search"
```

Comment:
Seems to be a bug on iPhone X with empty search table view 
<img width="263" alt="screen shot 2017-11-18 at 11 15 35 pm" src="https://user-images.githubusercontent.com/14966713/32988333-d86f655e-ccb6-11e7-80be-35236256c4f9.png">